pipelines/binary-builder.yml

```diff
@@ -26,116 +27,101 @@ resources: #####################################################################
   - name: builds-out
     type: git
     source:
-      uri: git@github.com:cloudfoundry/public-buildpacks-ci-robots
+      uri: git@github.com:SUSE/cf-public-buildpacks-ci-robots
       branch: master
       paths: [ binary-builds/* ]
       private_key: {{public-buildpacks-ci-robots-private-key}}
 
-<% dep_build_names = %w(bower go bundler composer dotnet glide godep httpd hwc jruby php php7 python node nginx ruby yarn) %>
 <% dep_build_names.each do |dep| %>
   - name: <%= dep %>-builds
     type: git
     source:
-      uri: git@github.com:cloudfoundry/public-buildpacks-ci-robots
+      uri: git@github.com:SUSE/cf-public-buildpacks-ci-robots
       branch: master
       paths: [ binary-builds/<%= dep %>-builds.yml ]
       private_key: {{public-buildpacks-ci-robots-private-key}}
 <% end %>
 
-<% dep_built_output_names = dep_build_names + %w(dotnet-framework) %>
```

Comment:
Not sure if we have a final decision to skip this buildpack. It is for running .NET apps on Linux, not the IIS / ASP.NET buildpack for running on Windows cells.